CHANGELOG.rst

```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
 
-- snap.model is now accessible at snap.core.model
+Removed Tag Prop Lifting Without Tag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- no default values anymore
+What changed
+    The ability to lift nodes by the presence of a tag property *without* specifying the tag name has been removed.  Given a tag property of "confidence", the ``#:confidence`` and ``#:confidence>90`` style syntax are no longer valid.  However, lifting by tag property *with* the tag, such as ``#foo.bar:confidence`` and ``#foo.bar:confidence>90`` remain valid.
 
-- snap methods nodesby are now different
+Why make the change
+    The necessary indexing to provide a performant way to lift nodes by the tag property without the tag is too expensive for the analytically dubious use case.
 
-- auth.getUserByName now async (why?)
+What you need to do
+    Any instances of lifting nodes by tag property without the tag will need to be updated to include the tag name.
 
-- user.admin -> user.isAdmin()
+Removed Insecure Mode
+~~~~~~~~~~~~~~~~~~~~~
+
+What changed
+    The "insecure" option in cell.yaml has been removed.
+
+Why make the change
+    Insecure mode of operation was a vestigial option originally designed to aid in bootstrapping and setting up initial admin users.  Telepath now allows for ``cell://`` and ``unix://`` connection schemes that can bypass authentication for local users making insecure mode unnecessary.
+
+What you need to do
+    If you have services deployed in insecure mode, they will need to be transitioned to using proper authentication.
+
+Removed Default Values From Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+What changed
+    Model properties may no longer have default values.
+
+Why make the change
+    The root reason for this change is a complex cascade of requirements which hinge on the simple concept of populating a default value.  In Synapse ``0.2.x``, nodes may be created and edited without lifting them.  This means that ingest speeds can be significantly increased by taking an "upsert" approach.  However, it also has the side effect of making it very difficult to know if a given node already has a value specified in another layer without lifting and fusing the node from all the properties in all the layers within the view.  Ultimately, by removing the expectation of default values for a given property, we have been able to allow the Cortex to create nodes without needing to lift them, creating a large performance benefit.
+
+What you need to do
+    If you have custom model elements that have default values, they will no longer be populated by default.  As a work around, you may create a trigger which populates the property when the node is added, but use caution when merging properties from multiple layers when populating defaults.
+
+Additional Changes
+------------------
+
+- map_async is now enabled by default for all slabs
+- Synapse tools may not be used across minor versions.
```

Comment:
we should enumerate this a bit more.

Reply:
(Our minor version is really a major version)

Reply:
agreed, i'll make it specific to using tools to connect to services of a different minor version.